zebra-rpc/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zebra-rpc"
-version = "3.0.0"
+version = "3.0.1"
```

Comment:
I’m not entirely sure, and it might not matter much in practice, but the changes described in the CHANGELOG look more like a minor release than a patch, according to SemVer.

> - Populated `asm` field returned by Zebra's RPC methods with code in script outputs as well as script types ([#10019](https://github.com/ZcashFoundation/zebra/pull/10019))

Populating an existing field introduces new observable behavior, which is considered new functionality under SemVer.

> - Republicized `valid_addresses` method ([#10021](https://github.com/ZcashFoundation/zebra/pull/10021))

Making a method public also adds new functionality.

Reply:
Bumped minor version in https://github.com/ZcashFoundation/zebra/pull/10088/commits/7c36471ad9aef254b85a566c171422c084fa2bf3.